eng/jacoco-test-coverage/pom.xml

```diff
@@ -183,7 +183,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
-      <version>4.46.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+      <version>4.45.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
```

Comment:
are we reducing version? is it intended?

Reply:
After the release process completes, a bot updates the version of the package to that of the next in-band release. `azure-cosmos` packages's current version is `4.45.0` which the bot upgraded to `4.46.0-beta.1`. This branch is for a hotfix release, so the intended version is `4.45.1`.